cms/core/wagtail_hooks.py

```diff
@@ -35,3 +43,18 @@ def global_admin_css() -> str:
 
 register_snippet(ContactDetailsViewSet)
 register_snippet(GlossaryViewSet)
+
+
+@hooks.register("after_edit_page")
+def after_edit_page(request: "HttpRequest", page: "Page") -> None:
+    if page.locale.language_code != settings.LANGUAGE_CODE:
+        return
+
+    # Check if proper translations of the page exist, which are not simple aliases
+    proper_translations = [
+        translation for translation in page.get_translations().only("alias_of") if not translation.alias_of
+    ]
+    if len(proper_translations) > 0:
```

Comment:
I believe this could be simplied as:

```suggestion
    has_proper_translations = page.get_translations().filter(alias_of__isnull=True).exists()
    if has_proper_translations:
```